src/datacatalog_tag_manager/tag_datasource_processor.py

```diff
@@ -27,18 +27,43 @@ def create_tags_from_csv(self, file_path):
 
         logging.info('')
         logging.info(f'Creating the Tags...')
-        created_tags = self.__create_tags_from_dataframe(dataframe)
+        created_tags = self.__execute_datacatalog_tags_method_on_dataframe(
+            dataframe, datacatalog_tags_method=self.__datacatalog_facade.create_or_update_tag)
```

Comment:
Do we need `datacatalog_tags_method=`? Since it's mandatory, I believe suppressing it makes the code clearer.